oktapam/data_source_secrets_test.go

```diff
@@ -0,0 +1,143 @@
+package oktapam
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/attributes"
+)
+
+func TestAccDatasourceSecretList(t *testing.T) {
```

Comment:
should we test the pagination somewhere?

Reply:
Based on the response to the comment above, my preference would be to move that into the SDK and not have to worry about things here.